src/ui/components/ProtocolViewer.tsx

```diff
@@ -40,7 +40,7 @@ type ChunkReducer = {
 };
 
 const chunkedRequests = (requests: RequestSummary[]): RequestSummaryChunk[] => {
```

Comment:
This is an example of a thing I wish we had inline documentation for. What is a "chunked request"? What does it mean to be "chunked"?

Is it merging protocol requests that have certain overlapping properties into a single object with a count that signifies how many separate requests were merged into the single one?